### PR TITLE
Chat: describe AI Integration demo

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 1]
```diff
@@ -1 +1,17 @@
-Chat is an interactive interface that allows users to send and receive messages in real time.
+This demo shows how to add an AI service to the Chat component to create a chatbot.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
This demo uses an AI service and the DevExtreme Chat component to create a chatbot UI.
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+## Handling dataSource (reloadOnChange: false)
+
+In this demo, a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/) is implemented (with custom load and insert functions). This store serves as a [dataSource](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) for the Chat component. [reloadOnChange](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#reloadOnChange) is set to `false`, which affects how messages are processed: updates are pushed directly into the store. See the [onMessageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event handler and `processMessageSending` function for more details.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
The Chat component's [dataSource](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) is a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/) that implements its own load and insert functions. The Chat deactivates [reloadOnChange](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#reloadOnChange) to push updates directly into the store and update the conversation manually. See the [onMessageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event handler and the `processMessageSending` function to review the code that manages data transfer between the Chat and its data store.
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+
+## Customizing Messages with messageTemplate
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
## Custom Message Template
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 4]
```diff
@@ -1 +1,17 @@
...
+
+The bot's messages feature "Copy" and "Regenerate" buttons. To add these buttons, implement a [messageTemplate](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#messageTemplate).
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
The Chat specifies a [messageTemplate](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#messageTemplate) that displays "Copy" and "Regenerate" buttons in bot messages.
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 5]
```diff
@@ -1 +1,17 @@
...
+
+## Converting Markdown Responses to HTML
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
## Response Format Conversion: Markdown to HTML
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 6]
```diff
@@ -1 +1,17 @@
...
+
+The AI model outputs responses in Markdown. This examples converts them to HTML with the [unified](https://github.com/unifiedjs) plugin library. See `convertToHtml` for more information.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
The AI model outputs responses in Markdown, while the Chat requires HTML output. This examples uses the [unified](https://github.com/unifiedjs) plugin library to convert response content. Review `convertToHtml` function code for implementation details.
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 7]
```diff
@@ -1 +1,17 @@
...
+
+## Changing Default Texts with Localization
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
## Default Caption Customization
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 8]
```diff
@@ -1 +1,17 @@
...
+
+Some UI texts are modified in the empty view with [localization](/Documentation/Guide/Common/Localization/) messages. You can apply a similar method to alter text.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md]:
```suggestion
The Chat component in this demo displays modified captions when the conversation is empty. The demo uses [localization](/Documentation/Guide/Common/Localization/) techniques to alter built-in text.
```

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md [hunk 9]
```diff
@@ -1 +1,18 @@
-Chat is an interactive interface that allows users to send and receive messages in real time.
+This demo uses an AI service and the DevExtreme Chat component to create a chatbot UI.
+
+## Handling dataSource (reloadOnChange: false)
+
+The Chat component's [dataSource](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) is a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/) that implements its own load and insert functions. The Chat deactivates [reloadOnChange](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#reloadOnChange) to push updates directly into the store and update the conversation manually. See the [onMessageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event handler and the `processMessageSending` function to review the code that manages data transfer between the Chat and its data store.
```
Comment (dirk-pieterse) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md:5]: `into the store` Do you mean 'storage'?

**Review by dirk-pieterse — APPROVED** [review]

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md (hunk 9 above)
Comment (vladaskorohodova) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md:5]: No, I mean the data store (an entity in DevExtreme data layer), like the CustomStore mentioned.

Comment (dirk-pieterse) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md:5]: Okay, thanks

**Pull request merged**